chrome/fed-modules.json

```diff
@@ -440,9 +440,34 @@
             }
         ]
     },
+    "rhosak": {
+        "manifestLocation": "/apps/rhosak/fed-mods.json",
+        "analytics": {
+            "APIKey": "eGAB8pv9DFgdj80STEaeHSMvRPAjdDUH"
+        },
+        "modules": [
+            {
+                "id": "rhosak",
+                "module": "./RootApp",
+                "routes": ["/application-services/streams"]
+            }
+        ]
+    },
+    "srs": {
+        "manifestLocation": "/apps/srs-ui-build/fed-mods.json",
+        "modules": []
+    },
+    "ar": {
+        "manifestLocation": "/apps/sr-ui-build/fed-mods.json",
```

Comment:
Just a question, the key has `ar`, but it has `sr-ui-build`. Isn't it a missclick?